components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -226,13 +221,8 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
                     <div className="flex flex-col mt-4 p-4 rounded-xl bg-gray-50 dark:bg-gray-800">
                         <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Balance Used</div>
                         <div className="mt-1 text-xl font-semibold flex-grow">
-                            <span className="text-gray-900 dark:text-gray-100">
-                                {typeof currentUsage === "number" ? Math.round(currentUsage) : "?"}
-                            </span>
-                            <span className="text-gray-400 dark:text-gray-500">
-                                {" "}
-                                / {usageLimit} Credit{usageLimit === 1 ? "" : "s"}
-                            </span>
+                            <span className="text-gray-900 dark:text-gray-100">{used}</span>
+                            <span className="text-gray-400 dark:text-gray-500"> / {totalAvailable} Credits</span>
```

Comment:
**thought**: Cross-posting for future reference, in case we'd like to separate these numbers to explain further how and which credit are carried over to the next month, how some credits could expire after some months, and more.

<img width="732" alt="Frame 541" src="https://user-images.githubusercontent.com/120486/203060593-78333e4c-d3d7-44cc-965d-9ca687ec062a.png">